app/src/examples/WithClampExample.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import Animated, {
+  useSharedValue,
+  useAnimatedStyle,
+  withSpring,
+} from 'react-native-reanimated';
+import { View, Text, Button, StyleSheet, ViewStyle } from 'react-native';
+
+const VIOLET = '#b58df1';
+const BORDER_WIDTH = 4;
+const FRAME_WIDTH = 300;
+const LOWER_CLAMP = 120;
+const UPPER_CLAMP = 220;
+const CLAMP_MARKER_HEIGHT = 40;
+
+function renderFramedExample(testedStyle: ViewStyle, description: string) {
+  return (
+    <React.Fragment>
```

Comment:
```suggestion
    <>
```

Reply:
Add an eslint rule?
``` diff
+     "react/jsx-fragments": ['error', 'syntax']
```

Reply:
Yes, but make sure that `<React.Fragment key={key}>` works (you can't use `key` with `<></>`).

Reply:
It works with key 💪